src/scss/custom/layout/_header-bottom.scss

```
@@ -9,6 +9,11 @@ $component-name: header-bottom;
 
   .navbar-brand {
     padding: 0;
+
+    .logo {
+      width: auto;
+      max-height: 50px;
+    }
```

```suggestion
  .navbar-brand {
    padding: 0;
    max-width: 250px;
    max-height: max(50px, 12vh);

    .logo {
      max-height: inherit;
    }
```
This should work nicely.